lib/node_modules/@stdlib/number/float64/base/to-words/test/test.assign.js

```diff
@@ -0,0 +1,279 @@
+	];
+
+	for ( i = 0; i < values.length; i++ ) {
+		v = values[ i ];
+		out = [ 0.0, 0.0 ];
+		w = toWords( v, out, 1, 0 );
+
+		t.equal( w, out, 'returns output array' );
+
+		// Convert to binary strings:
+		high = w[ 0 ].toString( 2 );
+		low = w[ 1 ].toString( 2 );
+
+		// Combine into a single bit literal:
+		if ( v > 0 ) {
+			str = '0'; // sign bit
+			str += lpad( high, 31, '0' );
+		} else {
+			str = lpad( high, 32, '0' );
+		}
+		str += lpad( low, 32, '0' );
+
+		expected = toBinaryString( v );
+		t.equal( str, expected, 'high+low equals bit string for ' + v );
+	}
+	t.end();
+});
+
+tape( 'if provided `NaN`, the sign bit may be either 0 or 1, the exponent must be all 1s, and the fraction cannot be all 0s (IEEE 754-1985)', function test( t ) {
+	var frac;
+	var high;
+	var low;
+	var out;
+	var w;
+
+	out = [ 0.0, 0.0 ];
+	w = toWords( NaN, out, 1, 0 );
+
+	t.equal( w, out, 'returns output array' );
+
+	// Convert to binary strings...
+	high = w[ 0 ].toString( 2 );
+	low = w[ 1 ].toString( 2 );
+
+	// Pad...
+	high = lpad( high, 32, '0' );
+	low = lpad( low, 32, '0' );
+
+	// Sign: +-1.
+	t.ok( high[0] === '1' || high[0] === '0', 'sign is either 1 or 0' );
+
+	// Exponent all 1s.
+	t.equal( high.substring( 1, 12 ), '11111111111', 'exponent all 1s' );
+
+	// Fraction cannot be all zeros.
+	frac = repeat( '0', 52 );
+
+	t.ok( high.substring( 12 ) !== frac.substring( 0, 32-12 ) || low !== frac.substring( 32-12 ), 'fraction not all 0s' );
+
+	t.end();
+});
+
+tape( 'if provided `+infinity`, the higher order word corresponds to a sign of 0, all 1s in the exponent, and all 0s in the fraction, while the lower order word is all 0s (IEEE 754-1985)', function test( t ) {
+	var high;
+	var low;
+	var out;
+	var w;
+
+	high = '0'; // sign
+	high += repeat( '1', 11 ); // exponent
+	high += repeat( '0', 20 ); // fraction
+	high = parseInt( high, 2 );
+
+	low = repeat( '0', 32 );
+	low = parseInt( low, 2 );
+
+	out = [ 0.0, 0.0 ];
+	w = toWords( PINF, out, 1, 0 );
+	t.equal( w, out, 'returns output array' );
+	t.equal( w[0], high, 'equals high word' );
+	t.equal( w[1], low, 'equals low word' );
+	t.end();
+});
+
+tape( 'if provided `-infinity`, the higher order word corresponds to a sign of 1, all 1s in the exponent, and all 0s in the fraction, while the lower order word is all 0s (IEEE 754-1985)', function test( t ) {
+	var high;
+	var low;
+	var out;
+	var w;
+
+	high = '1'; // sign
+	high += repeat( '1', 11 ); // exponent
+	high += repeat( '0', 20 ); // fraction
+	high = parseInt( high, 2 );
+
+	low = repeat( '0', 32 ); // fraction
+	low = parseInt( low, 2 );
+
+	out = [ 0.0, 0.0 ];
+	w = toWords( NINF, out, 1, 0 );
+	t.equal( w, out, 'returns output array' );
+	t.equal( w[0], high, 'equals high word' );
+	t.equal( w[1], low, 'equals low word' );
+	t.end();
+});
+
+tape( 'the function supports providing an output object (array)', function test( t ) {
+	var expected;
+	var out;
+	var w;
+
+	expected = toWords( pow( 2.0, 53 ), [ 0.0, 0.0 ], 1, 0 );
```

Comment:
```suggestion
	expected = toWords( pow( 2.0, 53 ), [ 0, 0 ], 1, 0 );
```

Reply:
Currently, this test is circular. At L226 and L227, we are effectively testing whether the function is isomorphic, not whether the function returns the expected values.